src/components/MyAccount/NewSettings/PhoneEmailForm.tsx

```diff
@@ -0,0 +1,305 @@
+import {
+  Icon,
+  TextInput,
+  Text,
+  Flex,
+  Button,
+  SkeletonLoader,
+  Form,
+} from "@nypl/design-system-react-components"
+import { useContext, useState } from "react"
+import { PatronDataContext } from "../../../context/PatronDataContext"
+import SaveCancelButtons from "./SaveCancelButtons"
+import SettingsLabel from "./SettingsLabel"
+import type { Patron } from "../../../types/myAccountTypes"
+
+interface PhoneEmailFormProps {
+  patronData: Patron
+  setIsSuccess: (boolean) => void
+  setIsFailure: (boolean) => void
+  inputType: "phones" | "emails"
+}
+
+const PhoneEmailForm = ({
+  patronData,
+  setIsSuccess,
+  setIsFailure,
+  inputType,
+}: PhoneEmailFormProps) => {
+  const isEmail = inputType === "emails"
+  const { getMostUpdatedSierraAccountData } = useContext(PatronDataContext)
+  const [inputs, setInputs] = useState(
+    isEmail
+      ? patronData[inputType]
+      : patronData[inputType]?.map((phone) => phone.number) || []
+  )
+  const [isLoading, setIsLoading] = useState(false)
+  const [isEditing, setIsEditing] = useState(false)
+  const [error, setError] = useState(false)
+
+  const [tempInputs, setTempInputs] = useState([...inputs])
+
+  const formUtils = {
+    regex: isEmail ? /^[^@]+@[^@]+\.[^@]+$/ : /^\+?[1-9]\d{1,14}$/,
+    labelText: `Update ${inputType}`,
+    addButtonLabel: isEmail ? "+ Add an email address" : "+ Add a phone number",
+    errorMessage: `Please enter a valid and unique ${
+      isEmail ? "email address" : "phone number"
+    }.`,
+    icon: `communication${isEmail ? "Email" : "Call"}`,
+    inputLabel: isEmail ? "Email" : "Phone",
+  }
+
+  const validateInput = (currentInput, inputs) => {
+    const isInputUnique =
+      inputs.filter((input) => input === currentInput).length === 1
+    return formUtils.regex.test(currentInput) && isInputUnique
```

Comment:
Can you verify this works? I'm able to submit these types of phone numbers:
<img width="365" alt="Screenshot 2024-11-05 at 9 52 05 AM" src="https://github.com/user-attachments/assets/5528a5b5-12c8-4367-be47-85bca1d84d76">

Reply:
Yeah this is the level of validation on prod and accepted by Sierra– any number basically

Reply:
Can bring this up with Sean if we want to have stricter validation?

Reply:
Yes, let's bring it up at standup

Reply:
Offline confirmed people can enter any old number for now